manifests/05-operator.yaml

```diff
@@ -30,5 +30,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: IMAGE
+          value: docker.io/openshift/origin-console:latest
```

Comment:
This is how other operators are handling this.

Reply:
Weird, but OK